Index.html

```diff
@@ -77,5 +77,170 @@ <h1>I'm Kerlos Baha.<br />Glad to see <br />you!</h1>
+        src="/picture/Snapshoot Portfolio2.png"
+        alt=""
+      />
+      <p class="repeated-p1">
+        Multi-Post <br />
+        Stories
+      </p>
+      <ul class="repeated-ul">
+        <li class="repeated-li1">CANOPY</li>
+        <li class="repeated-li2">Back End Dev</li>
+        <li class="repeated-li3">2015</li>
+      </ul>
+      <p class="repeated-p2">
+        A daily selection of privately personalized reads; no accounts or
+        sign-ups required.
+      </p>
+      <ul class="repeated-ul2">
+        <li class="repeated-li4">HTML</li>
+        <li class="repeated-li5">CSS</li>
+        <li class="repeated-li6">JavaScript</li>
+      </ul>
+      <button class="see-button">See project</button>
+    </section>
+
+    <!-- third page -->
+    <div class="space-2"></div>
+    <section class="repeated-section">
+      <img
+        class="repeated-img"
+        src="/picture/Snapshoot Portfolio3.png"
+        alt=""
+      />
+      <p class="repeated-p1">Tonic</p>
+      <ul class="repeated-ul">
+        <li class="repeated-li1">CANOPY</li>
+        <li class="repeated-li2">Back End Dev</li>
+        <li class="repeated-li3">2015</li>
+      </ul>
+      <p class="repeated-p2">
+        A daily selection of privately personalized reads; no accounts or
+        sign-ups required.
+      </p>
+      <ul class="repeated-ul2">
+        <li class="repeated-li4">HTML</li>
+        <li class="repeated-li5">CSS</li>
+        <li class="repeated-li6">JavaScript</li>
+      </ul>
+      <button class="see-button">See project</button>
+    </section>
+    <!-- four page -->
+    <div class="space-2"></div>
+    <section class="repeated-section-2">
+      <img
+        class="repeated-img"
+        src="/picture/Snapshoot Portfolio4.png"
+        alt=""
+      />
+      <p class="repeated-p1">
+        Multi-Post <br />
+        Stories
+      </p>
+      <ul class="repeated-ul">
+        <li class="repeated-li1">CANOPY</li>
+        <li class="repeated-li2">Back End Dev</li>
+        <li class="repeated-li3">2015</li>
+      </ul>
+      <p class="repeated-p2">
+        A daily selection of privately personalized reads; no accounts or
+        sign-ups required.
+      </p>
+      <ul class="repeated-ul2">
+        <li class="repeated-li4">HTML</li>
+        <li class="repeated-li5">CSS</li>
+        <li class="repeated-li6">JavaScript</li>
+      </ul>
+      <button class="see-button">See project</button>
+    </section>
+    <div class="space"></div>
+
+    <!-- about myself -->
+
+    <section class="about-myself">
+      <div class="div-flex">
+        <h1>About Myself</h1>
+        <p class="intro">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website. Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don’t
+          hesitate to contact me.
+        </p>
+        <p class="Connect">LET'S CONNECT</p>
```

Comment:
Kindly update this element to use a `<hx>` tag as it is a sub-title

Reply:
Fixed.